### PR TITLE
Adds scheduled rake task for cleaning up backup directories.

### LINKED ISSUE #535: cron job to remove week-old cdxj indexes from cdxj_backup
See description of original problem with openwayback's cdx files in #44.

In `/web-archiving-stacks/data/indexes/cdxj_backup` there are druid directories with per-WARC cdxj indexes that are created via the cdxj-merge step in wasCrawlDissemination. https://github.com/sul-dlss/was_robot_suite/blob/b0daaa9df62f07453ae1013cda885b0c1d74c98e/lib/dor/was_crawl/cdxj_merge_service.rb#L75

If the cdxj file in the druid dir is older than a week, it can be deleted, and if there are no other files, the druid directory can be deleted. 

Could be done by writing a rake task and scheduling it, or another approach that seems in line with what we usually do.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #535

## Why was this change made? 🤔
Good file hygiene.

## How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from other services or writing to shared file systems, test that web archive seed and crawl accessioning works properly in [stage|qa] environment, in addition to specs. ⚡

Rake tasks tested on QA.

### CODE REVIEW

File: lib/tasks/index.rake [hunk 1]
```diff
@@ -7,6 +7,8 @@ require "#{File.dirname(__FILE__)}../../../config/boot" # for Dor::Config settin
 # Location of the CDXJ index files used by pywb
 INDEX_DIR_CDXJ = Pathname.new(File.dirname(Settings.was_crawl_dissemination.main_cdxj_file)).freeze
 
+ONE_WEEK = 7 * 24 * 60
```
Comment (mjgiarlo) [inline lib/tasks/index.rake:10]: Cruft?

**Review by mjgiarlo — APPROVED** [review]
Approve pending cruft comment consideration

**Comment by lwrubel** [comment]
These druid directories are not typically empty. They contain a file ending in .cdxj, which is an index for a particular WARC that went through the cdxj indexing steps. That file is no longer needed to be retained after a week, we've determined.

**Comment by lwrubel** [comment]
I can see how what I described in the ticket was confusing. I was meaning to account for a situation where WARCs were somehow added to a druid a few days after the initial indexing, leading to files of different ages. We only wanted to get rid of those a week or more old.

**Comment by lwrubel** [comment]
Deleting my previous comments since I now understand this task is deleting the files, then the directories.

**Review by lwrubel — APPROVED** [review]

File: lib/tasks/index.rake (hunk 1 above)
Comment (justinlittman) [reply lib/tasks/index.rake:10]: Yes. Removing.

**Pull request merged**